charts/kueue/values.yaml

```diff
@@ -6,7 +6,7 @@ fullnameOverride: ""
 
 # Enable each function, like kustomize https://github.com/kubernetes-sigs/kueue/blob/main/config/default/kustomization.yaml
 enablePrometheus: false
-enableCertManager: true
+enableCertManager: false
```

Comment:
it should be enabled by default

Reply:
Please revert

Reply:
Uhm.... maybe I'm confused with the variable name.

By default, the internal cert manager should be used. We do this for plain yamls (not helm).

Reply:
For internal certs are enough to setup secret. It is always done without restrictions. But If we want to use a third-party one we just enable ```enableCertManager``` option or uncomment certmanager components in kustomize

Reply:
oh ok, so this is about the external cert manager.
`false` by default sounds good then :)
Can you update the README to say `cert-manager.io`. Hopefully that's more explicit.